string file_path
---
string graph_json
string node_json